### PR TITLE
feat(Footer): implements design

### PR DESCRIPTION
**Pull request by YgRamos9**
## Summary

- add typography
- add footer section
- add fonts mixies
- add icons

## Screenshots

image

## Known Issues

- links are not added

## Trello Card

- https://trello.com/c/jCZcsjaS/4-layout-create-main-search-footer-component

### CODE REVIEW

File: src/app/components/footer/footer.component.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<section class="footer-section">
```
Comment (andres-acc) [inline src/app/components/footer/footer.component.html:1]: Use the footer tag.
```suggestion
<footer class="footer-section">
```

File: src/app/components/footer/footer.component.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  <div class="left-section">
+    <div class="elements-container">
+      <span class="logo-wolox"></span>
+    </div>
+    <div class="elements-container">
+      <div class="social-media">
+        <span class="linked-in social-icon"></span>
+        <span class="social-text">Conecta con nostros</span>
+      </div>
+    </div>
```
Comment (andres-acc) [inline src/app/components/footer/footer.component.html:11]: Check if you really need the `elements-container` wrapper 

```suggestion
      <span class="logo-wolox"></span>
      <div class="social-media">
        <span class="linked-in social-icon"></span>
        <span class="social-text">Conecta con nostros</span>
      </div>
```

File: src/app/components/footer/footer.component.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<section class="footer-section">
+  <div class="left-section">
+    <div class="elements-container">
+      <span class="logo-wolox"></span>
```
Comment (andres-acc) [inline src/app/components/footer/footer.component.html:4]: Why don't use an ``?

File: src/app/components/footer/footer.component.scss [hunk 4]
```diff
@@ -0,0 +1,86 @@
+@import '/src/scss/colors';
+@import '/src/scss/mixins/text';
+@import '/src/scss/sizes';
+
+.footer-section {
+  align-items: center;
+  background: $alabaster;
+  bottom: 0%;
```
Comment (andres-acc) [inline src/app/components/footer/footer.component.scss:8]:
```suggestion
  bottom: 0;
```

File: src/app/components/footer/footer.component.scss [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  position: absolute;
+  width: $full-size;
+
+  .elements-container {
+    border-right: 1px solid $black-border-footer;
+    display: flex;
+    padding: 0 16px;
+  }
+
+  .floating-images {
```
Comment (andres-acc) [inline src/app/components/footer/footer.component.scss:21]:
```suggestion
  .floating-image {
```

File: src/app/components/footer/footer.component.scss [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0 26px;
+  }
+
+  .left-section {
+    display: flex;
+  }
+}
```
Comment (andres-acc) [inline src/app/components/footer/footer.component.scss:86]: Use the scss nesting structure.

**Review by andres-acc — CHANGES_REQUESTED** [review]

File: src/app/components/footer/footer.component.html [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    </div>
+    <div class="elements-container">
+      <div class="social-media">
+        <span class="linked-in social-icon"></span>
+        <span class="social-text">Conecta con nostros</span>
```
Comment (miguelhenaoa) [inline src/app/components/footer/footer.component.html:9]: Typo, `Conecta con nosotros`

**Review by miguelhenaoa — COMMENTED** [review]
Great job, review the Andres comments

**Review by andres-acc — APPROVED** [review]
Well done 👍

**Pull request merged**